services/core-web/src/components/mine/ExplosivesPermit/MineExplosivesPermitTable.tsx

```diff
@@ -471,7 +477,7 @@ const MineExplosivesPermitTable: FC<RouteComponentProps & MineExplosivesPermitTa
     <CoreTable
       condition={isLoaded}
       dataSource={transformRowData(data)}
-      rowKey={(record: IExplosivesPermit) => record.explosives_permit_guid}
+      rowKey={(record: MineExplosivesColumnType) => record.explosives_permit_guid}
```

Comment:
I see that we're actually setting the key up above- not necessary, but you could totally remove the rowKey attribute if you wanted to.